boards/arm/efm32pg_stk3402a/doc/efm32wg_stk3800.rst

```diff
@@ -0,0 +1,168 @@
+.. _efm32pg_stk3402a:
+
+EFM32PG-STK3402A
```

Comment:
Would a more friendly title be, "EFM32 Pearl Gecko Starter Kit" ?

Reply:
Done.